features/google/src/model/importers/activities-importer.js

```diff
@@ -43,10 +43,10 @@ class ActivityParser {
 }
 
 export default class ActivitiesImporter {
-    async import({ zipFile, googleAccount }) {
+    async import({ zipFile, facebookAccount: googleAccount }) {
```

Comment:
Now this is confusing.

Reply:
From a comment above. In polyImport it is still called facebookAccount, need to change it there, but not in this PR